documentation-site/components/theming/borders.js

```diff
@@ -8,42 +8,50 @@ LICENSE file in the root directory of this source tree.
 
 import * as React from 'react';
 import {useStyletron} from 'baseui';
-import {Property} from './common.js';
+import {Property, PropertyCompareTheme} from './common.js';
 
-export function Border({value}: {value: string}) {
-  const [css, theme] = useStyletron();
+export function Border({name}: {name: string}) {
+  const [css] = useStyletron();
   return (
-    <Property
-      title={value}
-      value={[
-        theme.borders[value].borderStyle,
-        theme.borders[value].borderWidth,
-        theme.borders[value].borderColor,
-      ]}
-    >
-      <div
-        className={css({
-          borderTopStyle: theme.borders[value].borderStyle,
-          borderTopWidth: theme.borders[value].borderWidth,
-          borderTopColor: theme.borders[value].borderColor,
-        })}
-      ></div>
-    </Property>
+    <PropertyCompareTheme
+      name={name}
+      concern="borders"
+      renderBox={({mode, commonStyles}) => (
+        <div
+          className={css({
+            ...commonStyles,
+            ...mode.borders[name],
```

Comment:
`mode` seems to be the `theme` obj here, correct? Why don't we keep the `theme` naming here to avoid any confusion?

Reply:
I need to distinguish between the theme for the doc site that is provided by `useStyletron` and `styled` and the fixed theme object that I import and pass around (`LightTheme`, `DarkTheme`). In some cases they are both in the same scope so I need two different names. Open to suggestions on what to call them though.

Reply:
`docTheme` and `previewTheme` maybe?

Reply:
`previewTheme` sounds good 👍

Reply:
cool - updated